### PR TITLE
Add GE 26931 Smart Motion Switch

### PR DESCRIPTION
**Pull request by thinkelastic**

### CODE REVIEW

**Comment by nechry** [comment]
no link with manufacturer_specific.xml file ?

**Comment by thinkelastic** [comment]
@nechry This is what I used:

http://products.z-wavealliance.org/Products/2035/XML
http://products.z-wavealliance.org/products/2035/configs

Let me know if there is anything else that is needed.

**Comment by techgaun** [comment]
@thinkelastic @nechry meant to say that you need to put an entry in https://github.com/OpenZWave/open-zwave/blob/master/config/manufacturer_specific.xml so that there's a reference for your config file. An example is this: https://github.com/OpenZWave/open-zwave/blob/master/config/manufacturer_specific.xml#L13

**Comment by thinkelastic** [comment]
@nechry @techgaun Thanks!

I added the reference to the file.

File: config/manufacturer_specific.xml [hunk 1]
```diff
@@ -537,6 +537,7 @@
 		<Product type="4944" id="3033" name="12729 3-Way Dimmer Switch" config="ge/12724-dimmer.xml"/>
 		<Product type="4944" id="3034" name="12730 Fan Control Switch" config="ge/12724-dimmer.xml"/>
 		<Product type="4944" id="3038" name="14294 In-Wall Smart Dimmer" config="ge/14294-dimmer.xml"/>
+		<Product type="494d" id="3032" name="26931 Smart Motion Switch" config="ge/26931-motion-switch.xml"/>
```
Comment (thinkelastic) [inline config/manufacturer_specific.xml]: Just updated the id  and validated using home assistant:

manufacturer_name: GE
product_name: Unknown: type=494d, id=3032

**Comment by thinkelastic** [comment]
I ran make xmltest:

open-zwave/config/ge/26931-motion-switch.xml validates

**Pull request merged**